src/test/java/hudson/plugins/active_directory/docker/TheFlintstonesIT.java

```diff
@@ -74,16 +123,13 @@ public void dynamicSetUp() throws Exception {
     }
 
     public void dynamicSetUp(boolean requireTLS) throws Exception {
-        dockerIp = requireTLS ? docker.getHost() : "dc1.samdom.example.com";
```

Comment:
this was just funky - it would use "localhost" as the server to connect to for TLS...

And yet the TLS cert for the samba server is not valid for "localhost" and caused a certificate validation exception.
As we now map `dc1.samdom.example.com` to a the docker host we will connect to it correctly and the cert will come from samba and as such be presented as `dc1.samdom.example.com`

Not sure how this worked before - most likely a change in the JVM or a change in the contain that went unmissed as the tests where not running.  Did not investigate further.